packages/video-ad-sdk/src/adContainer/helpers/supportsSrcdoc.js

```diff
@@ -0,0 +1,3 @@
+const supportsSrcdoc = () => 'srcdoc' in document.createElement('iframe');
```

Comment:
is this being executed multiple times? if yes then you can check once

Reply:
👍